app/views/layouts/mailer.html.erb

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Should be `<%= I18n.locale %>` maybe?

Reply:
It's a W3C recommendation to have the tag, so it shuts up the pedantic warnings :-)
But actually, no we don't want our emails to (necessarily) be in the same language as the UI language, they're orthogonal. It's a can I/we deliberately kicked down the road when I did the wacko I18n branch. One that will prolly be painful when we revisit it.

Reply:
Actually it won't be that painful. When we have multiple languages for our outgoing e-mails we will have a "flag" menu in the preview page. How we're to know which language to send is anyone's guess, however.

Reply:
`email_lang = I18n.locales.sample`